specs/darwin/kernel_panics.table

```diff
@@ -0,0 +1,18 @@
+table_name("kernel_panics")
```

Comment:
If you remove the Darwin-specific diction can we use these concepts on Linux later on?